radis/test/io/test_hitemp.py

```diff
@@ -180,53 +180,54 @@ def test_fetch_hitemp_OH_vaex(verbose=True, *args, **kwargs):
     )
 
 
-@pytest.mark.needs_connection
-def test_fetch_hitemp_partial_download_CO2(verbose=True, *args, **kwargs):
-    """Test partial download of HITEMP CO2 database.
-
-    Good to test fetch_hitemp.
-    ``CO2-02_02500-03000_HITEMP2010.h5`` is only 20 MB so it can be run on online
-    tests without burning the planet too much 🌳
-
-    """
-    from os.path import basename
-
-    wmin = 2700
-    wmax = 3000
-
-    # Check that we won't download all database for a reduced range:
-    # ... This is done at the HITEMPDatabaseManager level
-    # ... unittest for this part:
-    ldb = HITEMPDatabaseManager(
-        name="HITEMP-CO2",
-        molecule="CO2",
-        engine="default",
-        local_databases="~/.radisdb/hitemp/",
-    )
-    local_files, _ = ldb.get_filenames()
-    relevant_file, file_wmin, file_wmax = keep_only_relevant(
-        local_files, wavenum_min=wmin, wavenum_max=wmax
-    )
-
-    assert len(relevant_file) == 1
-    assert basename(relevant_file[0]).startswith("CO2-02_02500-03000_HITEMP2010.")
-    assert file_wmin == 2500
-    assert file_wmax == 3000
-
-    # Now run the full function :
-
-    df, local_files = fetch_hitemp(
-        "CO2",
-        load_wavenum_min=2700,
-        load_wavenum_max=3000,
-        verbose=3,
-        return_local_path=True,
-    )
-
-    assert "HITEMP-CO2" in getDatabankList()
-
-    assert len(local_files) == 1
-    assert basename(local_files[0]).startswith("CO2-02_02500-03000_HITEMP2010.")
+# #This test is outdated since version 2024 of CO2 HITEMP. This is a single file of 6 GB now.
```

Comment:
We can do partial downloads soon, then we can add this test again

Reply:
We are not compatible with the HITEMP CO2 in its 2010 version. The only version we can download is the newest. Not hard to implement but not done yet.

Reply:
Suggested in #784 . Please close if you're happy :)